src/uikit.cs

```diff
@@ -9862,10 +9872,6 @@ interface UIResponder : UIAccessibilityAction, UIAccessibilityFocus
 		[Export ("updateUserActivityState:")]
 		void UpdateUserActivityState (NSUserActivity activity);
 
-		[iOS (8,0)]
-		[Export ("restoreUserActivityState:")]
-		void RestoreUserActivityState (NSUserActivity activity);
```

Comment:
mmm, we have allowed parameter name changes in the past, but since the `UIUserActivityRestoring` protocol just got added lets just rename  `userActivity` to `activity` to avoid the breaking change.

Reply:
Sure, also, braking change is probably a too "strong" adjective. API will change, not break ;)

Doing it nevertheless

Reply:
It is a breaking change because of named parameters for example if anyone is doing  RestoreUserActivityState (activity: x); it would not compile anymore, but yeah not very likely